packages/itwin/geo-tools/package.json

```
@@ -81,38 +80,43 @@
     "cpx2": "^4.2.2",
     "enzyme": "^3.4.0",
     "enzyme-to-json": "^3.3.4",
-    "eslint": "^8.41.0",
+    "eslint": "^9.18.0",
+    "eslint-config-prettier": "^10.0.1",
+    "eslint-plugin-import": "^2.31.0",
+    "eslint-plugin-react": "^7.37.4",
+    "eslint-plugin-unused-imports": "^4.1.4",
     "ignore-styles": "^5.0.1",
     "jsdom": "^11.12.0",
     "jsdom-global": "3.0.2",
     "mocha": "^10.2.0",
     "nyc": "^15.1.0",
     "raf": "^3.4.0",
-    "react": "^17.0.2",
-    "react-dom": "^17.0.2",
-    "react-redux": "^7.2.6",
+    "react": "^18.3.1",
+    "react-dom": "^18.3.1",
+    "react-redux": "^7.2.9",
     "react-test-renderer": "^17.0.2",
-    "redux": "^4.1.0",
+    "redux": "^4.2.1",
     "rimraf": "^3.0.2",
     "sinon": "^17.0.1",
     "sinon-chai": "^3.7.0",
     "source-map-support": "^0.5.6",
     "ts-node": "^7.0.1",
     "ts-sinon": "^2.0.1",
-    "typescript": "~4.6.0",
+    "typescript": "~5.7.2",
     "xmlhttprequest": "^1.8.0"
   },
   "peerDependencies": {
     "@itwin/appui-abstract": "^4.0.0",
-    "@itwin/appui-react": "^4.10.0",
-    "@itwin/components-react": "^4.10.0",
+    "@itwin/appui-react": "^5.0.0",
```

i think we should be able to support 4.x and 5 simultaneously for now since youre not using any new 4.10+ APIs

```suggestion
    "@itwin/appui-react": "^4.10.0 || ^5.0.0",
```